src/test/resources/unimplemented_go/controlflow/controlflow.go

```diff
@@ -0,0 +1,6 @@
+package controlflow;
+
+ensures res > 0
+func a() (res int) {
+    for {continue}
+}
```

Comment:
this file should parse but not verify. I think we need an annotation to express that this file is parseOnly

Reply:
This could be done by adding `// ##(--parseOnly)` e.g. after the package clause. Note however that the `Config.merge` functions need to be adapted to use the disjunction when merging the `should...` flags, e.g. `shouldParse = shouldParse || other.shouldParse`

Reply:
Alternatively, we could add a `GobraParserTests` unit test class in the `viper.gobra.parsing` package that correctly configures Gobra to only do parsing and not more with a dedicated set of input files. It should be possible that this unit test class extends `AbstractGobraTests` to reuse most code used in e.g. `GobraTests`